shared/common-adapters/popup-menu.desktop.js

```diff
@@ -11,24 +11,47 @@ class Menu extends Component<void, Props, void> {
     }
 
     const realCSS = `
-    .menu-hover:hover {
-      background-color: ${(this.props.style && this.props.style.hoverColor) || globalColors.blue4}
+    .menu-hover:hover { background-color: ${(this.props.style && this.props.style.hoverColor) || globalColors.blue4}; }
+    .menu-hover-danger:hover { background-color: ${globalColors.red}; }
+
+    .menu-hover .title { color: ${globalColors.black_75}; }
+    .menu-hover-danger .title { color: ${globalColors.red}; }
+    .menu-hover-danger:hover .title { color: ${globalColors.white}; }
+    .menu-hover-danger .subtitle { color: ${globalColors.black_40}; }
+    .menu-hover-danger:hover .subtitle { color: ${globalColors.white}; }
     }
     `
 
     return (
-      <Box style={{...stylesMenuCatcher, ...this.props.style}} onClick={() => this.props.onHidden()}>
+      <Box style={{...stylesMenuCatcher}} onClick={e => {
```

Comment:
props.style don't actually belong here. this needs to cover the entire screen basically so it's margins shouldn't be messed with